syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -368,10 +369,9 @@ def handle_func_command(cls, command):
 
         return response
 
-    def _get_response(cmd, args, kwargs):
-        """
-            Return the evaluation of the cmd string parameter
-        """
+    @staticmethod
+    @memorize
```

Comment:
What is the reason for this to have changed?

Reply:
Just a small optimization nothing essential here